pymongoexplain/explainable_collection.py

```diff
@@ -16,24 +16,29 @@
 from typing import Union, List, Dict
 
 import pymongo
+from pymongo.collection import Collection
 from bson.son import SON
 
 from .commands import AggregateCommand, FindCommand, CountCommand, \
     UpdateCommand, DistinctCommand, DeleteCommand, FindAndModifyCommand
 
 Document = Union[dict, SON]
 
+
 class ExplainCollection():
-    def __init__(self, collection):
-        self.collection = collection
+    def __init__(self, collection_object):
+        self.collection_object = collection_object
```

Comment:
I think "collection" is a better name. "collection_object" is a bit verbose for Python.

Reply:
Done.